blacktie/jatmibroker-xatmi/src/main/java/org/jboss/narayana/blacktie/jatmibroker/core/transport/hybrid/stomp/StompManagement.java

```diff
@@ -5,6 +5,7 @@
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.net.Socket;
+import java.nio.ByteBuffer;
```

Comment:
There are two test failures with these changes.
TestTPCall.test_tpcall_x_c_type() and TestTPCall.test_tpcall_x_common().

I think it might be related to appending the message.body. The two tests allocate a struct buffer.
```
public void test_tpcall_x_common()  {
        X_COMMON dptr = (X_COMMON) connection.tpalloc("X_COMMON", "deposit");

        dptr.setLong("acct_no", 12345678);
        dptr.setShort("amount", (short) 50);
}
public void test_tpcall_x_c_type() {
        X_C_TYPE aptr = (X_C_TYPE) connection.tpalloc("X_C_TYPE", "acct_info");

        aptr.setLong("acct_no", 12345678);
        aptr.setByteArray("name", "TOM".getBytes());

        float[] foo = new float[2];
        foo[0] = 1.1F;
        foo[1] = 2.2F;
        aptr.setFloatArray("foo", foo);

        double[] balances = new double[2];
        balances[0] = 1.1;
        balances[1] = 2.2;
        aptr.setDoubleArray("balances", balances);
}
```

SO I think it makes sense to use the ByteBuffer rather than StringBuffer to assemble the data.
How do you think about this ?